src/app/modules/main/profile_section/profile/module.nim

```diff
@@ -118,11 +135,22 @@ method storeProfileShowcasePreferences(self: Module,
     if acc.showcaseVisibility != ProfileShowcaseVisibility.ToNoOne:
       revealedAddresses.add(acc.address)
 
+  var verifiedTokens: seq[ProfileShowcaseVerifiedTokenPreference] = @[]
+  var unverifiedTokens: seq[ProfileShowcaseUnverifiedTokenPreference] = @[]
+
+  for asset in assets:
+    # TODO: more obvious way to check if it is verified or not
+    if asset.communityId == "":
```

Comment:
🙄 I would think the condition would be reversed here. What's the rational for having a verifiedToken add when the commnunityId is empty please?

Reply:
Yeah, it doesn't look very obvious, but we agreed on that terminology with wallet team. A verified token is a token that is recognized by Uniswap and other exchange services. Unverified tokens for now is a community tokens, but very soon i'm going to divide assets on two different models so this check will gone.